Source/Charts/Data/Interfaces/BarChartDataSetProtocol.swift

```diff
@@ -39,4 +39,13 @@ public protocol BarChartDataSetProtocol: BarLineScatterCandleBubbleChartDataSetP
     
     /// array of labels used to describe the different values of the stacked bars
     var stackLabels: [String] { get set }
+    
+    /// array of gradient colors [[color1, color2], [color3, color4]]
```

Comment:
I know this applies to many things in the protocol, but these properties are not necessary to create a data set for `BarChart`. The framework can provide an implementation in it's `BarChartDataSet`, but they shouldn't be required by the protocol.

Reply:
How will we access this property in the renderer without using the protocol?